core/storage/suggestion/gae_models.py

```diff
@@ -1292,3 +1292,642 @@ def export_data(
                     date.isoformat() for date in model.contribution_dates]
             }
         return user_data
+
+
+class TranslationReviewStatsModel(base_models.BaseModel):
+    """Records the contributor dashboard translation review stats. There
+    is one instance of this model per (language_code, reviewer_user_id,
+    topic_id) tuple.
```

Comment:
```suggestion
    """Records the translation review stats. There is one instance of 
    this model per (language_code, reviewer_user_id, topic_id) tuple.
```

I think "contributor dashboard" is unnecessary as there are other places where this model can theoretically be used. Also please add information about the structure of this model ID.

Reply:
Done